ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js

```diff
@@ -157,6 +155,15 @@ export function CatalogItemModalController($scope, $filter, blueprintService, pa
                 return `${$scope.config.name || $scope.config.symbolicName || 'Blueprint'} ${$scope.isUpdate() ? 'updated' : 'saved'}`;
         }
     };
+    $scope.getCatalogURL = () => {
+        switch ($scope.state.view) {
+            case VIEWS.form:
+                return '';
```

Comment:
return null? so it can be used in `ng-if="getCatalogURL()"`

Reply:
@algairim This url is not shown when the modal is in `form` state so its value is useless here. I can return `null` sure.

Reply:
`null` makes sense if you apply any sort of `if` condition to result of `getCatalogURL()`, either implicit or explicit. No need to change it otherwise.